src/catalog/src/builtin.rs

```diff
@@ -5947,20 +5947,62 @@ ON mz_internal.mz_sink_status_history (sink_id)",
     is_retained_metrics_object: false,
 };
 
+pub const MZ_SOURCE_STATISTICS_AGG_VIEW: BuiltinView = BuiltinView {
+    name: "mz_source_statistics_agg_view",
+    schema: MZ_INTERNAL_SCHEMA,
+    column_defs: None,
+    // everything but `params`
+    sql: "
+SELECT
+    id,
+    bool_and(snapshot_committed) as snapshot_committed,
+    SUM(messages_received) AS messages_received,
+    SUM(bytes_received) AS bytes_received,
+    SUM(updates_staged) AS updates_staged,
+    SUM(updates_committed) AS updates_committed,
+    SUM(envelope_state_bytes) AS envelope_state_bytes,
+    SUM(envelope_state_records) AS envelope_state_records,
+    -- Ensure we aggregate to NULL when not all workers are done rehydrating.
+    CASE
+        WHEN bool_or(rehydration_latency IS NULL) THEN NULL
+        ELSE MAX(rehydration_latency)
+    END AS rehydration_latency
```

Comment:
@teskje I believe this makes the `hydration_statuses` view strictly more correct, as it exposes the max rehydration time only after all workers have rehydrated

Reply:
Uff, indeed. I somehow missed that `mz_source_statistics` was per worker. So the current `mz_hydration_statuses` has the defect that it will contain sources that have multiple workers multiple times. Thanks for fixing this!

Btw, I think this `_agg_view` suffix does not have a precedent in our catalog naming so far. In compute, we name give per-worker relations a `_per_worker` suffix and use the suffix-less name for the aggregated relation. Could we do that here as well?

Reply:
Ah, I totally missed the rest of the discussion below. I'd still prefer the `_per_worker` suffix to `_raw`, because in Compute we use mostly `_raw` for relations that have counts encoded in the DD diff field. But I think there are also cases were we use it to just mean "you probably want to query a view on this instead", so I guess it still works here.